apps/web/pages/v2/apps/installed/[category].tsx

```diff
@@ -109,8 +110,10 @@ const IntegrationsList = ({ data }: { data: inferQueryOutput<"viewer.integration
                 installed
               />
             </div>
-          }
-        />
+          }>
+          {/* TODO: @hariom move this into /app-store/packages GeneralSettings */}
```

Comment:
same: hey @hariombalhara this would be awesome to be abstracted into something like "GeneralAppSettings"